src/HealthChecks.MySql/DependencyInjection/MySqlHealthCheckBuilderExtensions.cs

```diff
@@ -9,13 +10,16 @@ namespace Microsoft.Extensions.DependencyInjection;
 public static class MySqlHealthCheckBuilderExtensions
 {
     private const string NAME = "mysql";
+    internal const string HEALTH_QUERY = "SELECT 1;";
 
     /// <summary>
-    /// Add a health check for MySql databases.
+    /// Add a health check for MySQL/MariaDB Server databases.

```

Comment:
MariaDB?

Reply:
yes, there are the same thing (practically)